src/components/UI/Card/Card.jsx

```diff
@@ -27,6 +28,11 @@ const Card = (props) => {
     }
   };
 
+  //passing checkbox state and edit mode to modify the delete list
+  useEffect(() => {
+    props.addToDeleteList(props.id, isSelected, isEdited);
+  }, [isSelected, isEdited]);
```

Comment:
Please check for eslint warning:
![image](https://user-images.githubusercontent.com/52723192/149241699-2e7c9dd1-e842-4228-9d1e-415105d28a9e.png)